src/mpi/romio/adio/ad_lustre/ad_lustre_rwcontig.c

```diff
@@ -4,6 +4,13 @@
  */
 
 #include "ad_lustre.h"
+#ifndef _XOPEN_SOURCE
+#define _XOPEN_SOURCE 500
+#endif
+#ifndef _ISOC11_SOURCE
+#define _ISOC11_SOURCE 1
+#endif
```

Comment:
a comment noting which features these defines enable would be helpful for someone reading the code

Reply:
Apparently this was only needed for our cluster. Removed it.